### PR TITLE
Create a sample IWA directory containing a minimum viable IWA app

### PR DESCRIPTION
**Pull request by ycheng627**

### CODE REVIEW

File: diagnostics-extension/src/services/telemetry.ts [hunk 1]
```diff
@@ -76,6 +76,9 @@ export class TelemetryServiceImpl extends TelemetryService {
   async getOemData(): Promise<OemData> {
     return dpsl.telemetry.getOemData();
   }
+  async getCpuInfo(): Promise<CpuInfo> {
```
Comment (cebrusfs) [inline diagnostics-extension/src/services/telemetry.ts:79]: Curious, @1c3t3a is that expected that we didn't have APIs in example extensions?

File: sample-iwa/src/manifest.webmanifest [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Simple IWA", // Displayed name instead of isolated-app://gibberish URL
+  "id": "/",            // Must be "/"
+  "scope": "/",         // Must be "/"
+  "start_url": "/",     // Must be non-empty and refer to a resource in the bundle
+  "version": "0.1.0",   // Compulsory to be non-null.
+  "display": "standalone", // Should be one of the windowed ones
```
Comment (cebrusfs) [inline sample-iwa/src/manifest.webmanifest:7]: > Should be one of the windowed ones

I don't get this comment.

File: sample-iwa/src/manifest.webmanifest [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "icons": [               // Must have at least one valid entry
+    {
+      "src": "/icon.png", // Note, this file should also exist.
```
Comment (cebrusfs) [inline sample-iwa/src/manifest.webmanifest:10]: > Note, this file should also exist.

Suggest to use "have to" or "must" if it is a requirement.

**Review by cebrusfs — COMMENTED** [review]
Defer one minor question to @1c3t3a

File: diagnostics-extension/src/services/telemetry.ts [hunk 4]
```diff
@@ -76,6 +76,9 @@ export class TelemetryServiceImpl extends TelemetryService {
...
+    return dpsl.telemetry.getCpuInfo();
```
Comment (1c3t3a) [inline diagnostics-extension/src/services/telemetry.ts:80]: Should we bump the dps version in `package.json`? I am not 100% sure whether the current version supports CPU info.

File: sample-iwa/src/manifest.webmanifest [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      "type": "image/png",
+      "sizes": "512x512",
+      "purpose": "any maskable"
```
Comment (1c3t3a) [inline sample-iwa/src/manifest.webmanifest:13]: Maybe also add a comment on what this means?

File: sample-iwa/src/manifest.webmanifest (hunk 2 above)
Comment (1c3t3a) [reply sample-iwa/src/manifest.webmanifest:7]: Probably talking about how the app will be open, so as part of a browser window or as a standalone app with it's own window.

I would maybe write "The app should open as a standalone window"

File: sample-iwa/src/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+const extensionId = "gogonhoemckpdpadfnjnpgbjpbjnodgc";
+const request = {
+  type: 'telemetry',
+  telemetry: {
+    infoType: "cpu"
+  }
+}
+
+document.getElementById('hello-btn').addEventListener('click', event => {
+  window.chrome.runtime.sendMessage(
```
Comment (1c3t3a) [inline sample-iwa/src/index.js:10]: nit: Maybe put a small comment on what this does and the structure of a request?

File: sample-iwa/src/index.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (Lamzin) [inline sample-iwa/src/index.html:1]: Maybe put files under new `diagnostics-iwa` fodler to be consistent with other folders?

File: diagnostics-extension/src/services/telemetry.ts (hunk 1 above)
Comment (cebrusfs) [reply diagnostics-extension/src/services/telemetry.ts:79]: I see. Can I assume that this is just an example and we expect new users follow this example to add missing pieces by themselves. Then, I don't think we need to keep this example up to date with latest API.

**Pull request closed without merging**